### PR TITLE
Feature/282 short course bottom fe

### PR DESCRIPTION
**Pull request by nicklee**
All of the course code that hasn't been reviewed in one PR

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/molecules/key-details/key-details.html [hunk 1]
```diff
@@ -1,72 +1,107 @@
...
+        {% endif %}
+        {% if page.next_open_day_date %}
+            <div class="key-details__section key-details__section--open-days">
+                <h4 class="body body--two key-details__sub-heading">{{ programme_page_global_fields.key_details_next_open_day_title }}</h4>
+                <ul class="key-details__list">
+                    <li class="key-details__list-item">{{ page.next_open_day_date }}</li>
+                    <li class="key-details__list-item">
+                        <a class="key-details__link link link--secondary link--link" href="{{ page.link_to_open_days }}">
+                            <span class="link__label">{{ programme_page_global_fields.key_details_book_or_view_all_open_days_link_title }}</span>
+                            <svg width="12" height="8" class="link__icon"><use xlink:href="#arrow"></use></svg>
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/molecules/key-details/key-details.html]: hide this one too?

File: rca/project_styleguide/templates/patterns/molecules/key-details/key-details.html [hunk 2]
```diff
@@ -1,72 +1,107 @@
...
+                    <li class="key-details__list-item">{{ detail.text }}</li>
+                {% endfor %}
+                </ul>
+            </div>
+        {% endif %}
+        {% if page.programme_specification.url %}
+            <div class="key-details__section key-details__section--action">
+                <a class="key-details__link link link--secondary link--download" href="{{ page.programme_specification.url }}">
+                    <span class="link__label">Download the {{ page.title }} {{ page.degree_level }} programme specification</span>
+                    <svg width="8" height="12" class="link__icon"><use xlink:href="#download"></use></svg>
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/molecules/key-details/key-details.html]: and this one

File: rca/project_styleguide/templates/patterns/organisms/carousel/carousel--quotes.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            {% endfor %}
+        </ul>
+        <div class="carousel__controls-group grid" data-glide-el="controls">
+            <button class="carousel__button carousel__button--prev button" aria-label="Previous" data-glide-dir="<">
+                <svg class="carousel__button-icon" width="46" height="46">
+                    <use xlink:href="#arrow-left" />
+                </svg>
+            </button>
+            <button class="carousel__button carousel__button--next button" aria-label="Next" data-glide-dir=">">
+                <svg class="carousel__button-icon" width="46" height="46">
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/organisms/carousel/carousel--quotes.html]: i guess we can hide these one's too as you've got the aria-label on the button

**Comment by nicklee** [comment]
Polluted with staging, moving to backup

**Pull request closed without merging**